AppKit/CPTextView/CPTypesetter.j

```diff
@@ -319,7 +320,7 @@ var CPSystemTypesetterFactory,
         measuringRange.length++;
 
         var currentCharCode = theString.charCodeAt(glyphIndex),  // use pure javascript methods for performance reasons
-            rangeWidth = [theString.substr(measuringRange.location, measuringRange.length) sizeWithFont:currentFont inWidth:NULL].width + currentAnchor;
+            rangeWidth = sizingFunction(theString.substr(measuringRange.location, measuringRange.length), nil, currentFont, NULL).width + currentAnchor;
 
```

Comment:
For this function call to correct you need to pass the selector (in this case ```@selector(sizeWithFont:inWidth:)```) as the second argument instead of ```nil```. 

The reason for this is that a method implementation function looks like this. ```function(self, _cmd, arg1, arg2...)```. So if you send ```nil``` as the second argument ```_cmd``` will be ```nil```. ```_cmd``` is seldom used so it will work most of the time but will not be 100% correct.